src/RcsbSearch/SearchRequest.ts

```diff
@@ -17,16 +17,18 @@ export class SearchRequest {
         if(!this.fetch)
             throw "ERROR: fetch function was not provided"
     }
-    public async request(query: SearchQuery): Promise<QueryResult|null>{
+    public async request(query: SearchQuery, headers?: Object | null | undefined): Promise<QueryResult|null>{
         const localObj: QueryResult | null = LST.getItem<SearchQuery,QueryResult|null>(query);
+        const localHeaders : Object = headers || {};
```

Comment:
Is this a valid TypeScript? Basically in JavaScript I would do smth like:

```javascript

let headers = headers || {};

```

Reply:
You have to use `??` instead of `||`